packages/html/src/utils.ts

```diff
@@ -6,6 +6,72 @@ import * as domhandler from "domhandler";
 import * as domserializer from "dom-serializer";
 import { RenderOptions } from "./render.ts";
 
+/**
+ * Converts a React-style CSS object to a CSS string.
+ * Supports vendor prefixes, pixel value shorthand, and comprehensive CSS properties.
+ * @param styleObject - The style object with React-style camelCase properties
+ * @returns A CSS string suitable for the style attribute
+ */
+const styleObjectToCssString = (styleObject: Record<string, any>): string => {
```

Comment:
This function is duped between utils.ts and render.ts

Reply:
ups, fixed and moved utils.ts to test/

we should split up render.ts, but i'll do that in another PR